phpBB/styles/prosilver/template/posting_body.html

```diff
@@ -1,4 +1,7 @@
 
+<!-- INCLUDEJS jquery.caret.js -->
```

Comment:
I think for now S_PRIVMSGS will be fine. We can look later in the project on the exact condition.